### ISSUE TITLE
Old collector sets bug

### ISSUE THREAD

**Issue by TheBest0555** [ValveSoftware/Dota2-Gameplay#6935]
### Item name

A lot of them

### Description

I was selling old sets, and at the moment of transferring the set, I realized that it could not be transferred, since it was printed out. I went through the inventory and compiled a list of old sets that are not given away after printing.
Fractured Envoy - arc warden set
Trail of the Sanguine Spectrum - Bloodseeker set
Loaded Prospects - Brewmaster set
Fires of Vashundol - Doom set
Dread Ascendance - Doom set
Litany of the Damned - Doom set
Knight of the Burning Scale - set Dk
Third Awakening - set Dk
Fires of the Volcanic Guard - Ember spirit set
Astral Terminus - Enigma set
Arcane Inverter - Gyrocopter Set
Armor of the Unyielding Mask - Juggernaut Set
Nightsilver's Resolve - Luna Set
Darkheart Redemption - Lycan Set
Creed of the Skullhound - Lycan Set
Cruelties of the Spiral Bore - Magnus Set
Serpent of the Emerald Sea - medusa set
The Family Values - Meepo set
Pitmouse Fraternity - Meepo set
Legends of Darkheart Pursuit - Balanar set
Silence of the Starweaver - Oracle Set
Pitfall Crusader - Pangoler set
Creeping shadow - Phantom assasin set
Visions of the Lifted Veil - Phantom assasin Set
Diabolical Fiend - Shadow feind set
Stormwrought Arbiter - Sven Set
Indomitable Legacy - Sven Set
Primer of the Sapper's Guile - Techers set
Fortified Fabricator - Tinker set
Brawler of the Glacier Sea - Tuskar set
Raiments of the Obsidian Forge - Underlord Set
Dirgeful Overlord - Undying set
Echoes of the Eyrie - VengefuL set
Vision of the Seraph Scion - VengefuL set
Barding Of The Soul Keeper - Visage set
Dread Compact - Warlock Set
Flowersong Tempest - Windrunner set
Flight of the Sparrowhawk - Windrunner set
Morbific Provision - set Witch doctor
Stonemarch Sovereign - set Wraith king
Wartorn Heavens - Zeus set
My profile - https://steamcommunity.com/id/totkrasava/

### Example Match ID and Timestamp

-

**Comment by Aluerie** [comment]
The reality of these Collector Cache sets is, that after unbundling, they were never supposed to be giftable, piece by piece. 

The CC1 sets from latest Diretide BP are the ones "positively-bugged" where you could gift them piece by piece anyway. 

The community got confused when CC2 sets were not like that. "Why there are suddenly different laws for CC1 and CC2. I wouldn't have unbundled it if I knew" - a lot of complaints like this on Reddit/here were posted. 

In the end, it was recently ~~fixed~~ added positive-for-community bug for CC2 sets. 

--- 

So there is really no bugs in here. These olds collector cache sets are not supposed to be giftable or tradable after unbundling. You should probably close your issue.

**Comment by shaileshgarg** [comment]
https://github.com/ValveSoftware/Dota2-Gameplay/issues/6375, and they say I was wrong.

**Comment by TheBest0555** [comment]
> The reality of these Collector Cache sets is, that after unbundling, they were never supposed to be giftable, piece by piece.
> 
> The CC1 sets from latest Diretide BP are the ones "positively-bugged" where you could gift them piece by piece anyway.
> 
> The community got confused when CC2 sets were not like that. "Why there are suddenly different laws for CC1 and CC2. I wouldn't have unbundled it if I knew" - a lot of complaints like this on Reddit/here were posted.
> 
> In the end, it was recently ~fixed~ added positive-for-community bug for CC2 sets.
> 
> So there is really no bugs in here. These old (and current, and future) collector cache sets are not supposed to be giftable or tradable after unbundling. You should probably close your issue.

I do not think that this appeal is incorrect, but on the contrary, it requires a solution. If Valve did adopt a policy that after printing sets can still be donated, then this should be corrected everywhere.

**Comment by TheBest0555** [comment]
+ Desert Blood - Furion set
